pomegranate/NaiveBayes.pyx

```diff
@@ -320,3 +369,13 @@ cdef class NaiveBayes( object ):
                     y_ptr[i] = j
 
         return y
+
+    cdef numpy.ndarray _convert( self, data ):
```

Comment:
This probably shouldn't be a cdef function if you aren't using any C level functionality. We might also want to talk about factoring this to be a general utils.pyx function. Good implementation though.